### ISSUE TITLE
no notification after redis value change

### ISSUE THREAD

**Issue by Nexulo** [mebjas/realtime-notification#2]
Hello and thank you for the useful script!

I'm using the sample script and the connection to socket.io & redis works. 
But when i'm changing the test-key over redis-cli to a new value, i never got a notification of the changed value.

for example:

1. with following command, i set the „test“ key to „hello“: 
```
set test hello
OK
```

2. starting server.js
```
node server.js
[info] listening to port 8090
[info] redis server ready
[info] redis data server ready
```

3. set new value of the „test“ key:
```
set test "new value“
OK
```

4. i should receive a notification with the changed value but i never get one...?

**Comment by Nexulo** [comment]
Nobody an idea?

**Comment by mebjas** [comment]
@maisen20 
Yes, if your web client in connected to this test key on redis, it should get notified via a web socket event listener.
If that's not happening - there could be some version issue or some bug which we need to debug, can you share more information on the environment?

**Comment by mebjas** [comment]
@maisen20 updates?
